src/styles/globals.css

```
@@ -395,3 +463,18 @@
+
+/* Prevent iOS Safari auto-zoom on input focus (triggers when font-size < 16px) */
+@media screen and (max-width: 768px) {
+  input:not([type="checkbox"]):not([type="radio"]):not([type="range"]),
+  select,
+  textarea {
+    font-size: max(16px, 1em);
+  }
+}
+
+@custom-variant retina {
+  @media (-webkit-min-device-pixel-ratio: 2), (min-resolution: 2x) {
```

Stylelint (17.10.0)

[error] Vendor-prefixed media feature name (media-feature-name-no-vendor-prefix)

(media-feature-name-no-vendor-prefix)